lib/gem-miner/cli.rb

```diff
@@ -27,7 +32,21 @@ class CLI < Thor
       and the repositories that use the gem as an array of values.
     LD
     def gems(github_search_query)
-      puts GemMiner::Miner.gems_for(github_search_query, options[:token]).to_yaml
+      output GemMiner::Miner.gems_for(github_search_query, options[:token]).to_yaml
+    end
+
+    private
+
+    # Output the result of a command to either the terminal or a file,
+    # depending on the options given.
+    def output(result)
+      unless options[:output]
```

Comment:
Shouldn't matter. It's easier to read `if  x ... else ...`,  rather then `unless y ... else`